### ISSUE TITLE
Trace back Animated Vector Drawable to Shapeshifter

### ISSUE THREAD

**Issue by KaustubhPatange** [alexjlockwood/ShapeShifter#347]
Is this possible? Currently only SVG, Vector drawables are supported for import. Is there any chance for supporting feature of importing Animated Vector Drawable?

**Comment by alexjlockwood** [comment]
Yeah, it's definitely possible, and a valid feature request. Might even be not too hard to support AVDs that are expressed in a single XML file. AVDs that are split up into multiple files would be harder.

**Comment by imashnake0** [comment]
This sort of ties to not being able to save a session. Am I missing something or do we have to recreate the animation every time we close the tab?
